esp-wifi/MIGRATING-0.11.md

```diff
@@ -26,4 +26,8 @@ As part of limiting public API changes due to config options, the `csi_enabled`
 # In `Cargo.toml`:
 -esp-wifi = { version = "0.12.0", features = ["wifi"] }
 +esp-wifi = { version = "0.12.0", features = ["wifi", "csi"] }
-```
\ No newline at end of file
+```
+
+## Setting the power-save-mode is allowed now even when `coex` feature is enabled
+
+Previously we didn't allow to change the power-save-mode with the `coex` feature. This limitation is now lifted.
```

Comment:
The migration guide should detail code that needs to be changed (and IMO only the non-trivial of those changes, too). I don't think a new feature needs to be documented like this, but it should be part of the changelog.

Reply:
true, true